tslint.json

```diff
@@ -6,6 +6,7 @@
         "quotemark": [true, "single"],
         "trailing-comma": [false],
         "interface-over-type-literal": false,
-        "arrow-parens": false
+        "arrow-parens": false,
+        "variable-name": [true, "check-format", "allow-leading-underscore"]
```

Comment:
ts error:  variable is declared but never used.

ts compile allows leading underscore to suppress the warning. which is useful when the function signature is fixed. e.g.   aws express middleware.

express app.use requires  (req, resp, next) => {}
so i can write as (req, _resp, next) => {}